Automattic/simplenote-macos
# Share Popover Style

### Details:
This PR adjusts the Share Popover, so that it reacts to NSAppearance changes.

@etoledom Sir!! May I annoy you, yet again?
Thanks in advance!!

Ref. #458 

### Test
1. Launch Simplenote
2. Click over the top right Icon `( ... )` > `Collaborate`

- [x] Verify a Popover shows up at the bottom center of the screen
- [x] Verify that switching to Light / Dark theme, while the popover is onScreen immediately causes the Popover to refresh (and match the new style!)

### Release
These changes do not require release notes.
